src/resources/postcss/views/full/_event-status.pcss

```diff
@@ -0,0 +1,70 @@
+/* -----------------------------------------------------------------------------
+* Event Status.
+* ----------------------------------------------------------------------------- */
+
+.tribe-events-status__text {
```

Comment:
all the classes here that are not part of the event single should be wrapped in a `.tribe-events` class so that we are only targeting within our views.